src/components/BorrowerProfile/MoreAboutLoan.vue

```diff
@@ -46,7 +46,10 @@
 					>
 					</div>
 				</div>
-
+			</div>
+			<!-- "Follow" button for US loans links here from SummaryCard and LendCta -->
+			<a name="follow"></a>
```

Comment:
I had to add this anchor destination here, because the rest of this component hasn't loaded yet. This allows us to anchor link here even if the other elements are not in the dom yet